### PR TITLE
Additional Projects added, Contributors sections.

### PR DESCRIPTION
**Pull request by kd9lsv**
Note this is what has grown over the past month, so Please review and let me know if you want me to actually break it into different PRs. Doesn't hurt to ask.

# List of Added Items
Not1MM
OpenCwKeyerMK
Node-Red-Contesting-Dashboard
MorseRunner-CE
Ham Radio OS's 
TLF
FreeDATA
W0CHP-PiStar
Additional Codec2 applications

### CODE REVIEW

**Comment by DD5HT** [comment]
Nope it's fine, I take a look this weekend. Thank you a lot.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,4 +9,9 @@ Please ensure your pull request adheres to the following guidelines:
 + Check your spelling and grammar.
 + The pull request and commit should have a useful title.
 
-Thank you for your suggestions!
\ No newline at end of file
++ If unfamilar with Github and just wanting to add changes, please fill out the following form https://forms.gle/XX9zwJXgWZhVwk8k8 and the maintiners will review the proposed additions.
```
Comment (DD5HT) [inline CONTRIBUTING.md:12]: Do you think this is necessary or a workflow of creating an issue maybe easier?

File: README.md [hunk 2]
```diff
@@ -6,24 +6,35 @@
...
++ [G90Tools](https://github.com/OpenHamradioFirmware/G90Tools)  Tools and guides for working with the Xiegu G90 HF radio firmware
 + [OpenRTX](https://github.com/OpenRTX/OpenRTX) OpenRTX is a free and open source firmware for digital amateur radio devices, top-down designed with modularity, flexibility and performance in mind.
++ [K3ng CW Keyer](https://github.com/k3ng/k3ng_cw_keyer) is an open source Arduino based CW Keyer.
 
 ### Decoding
 
 + [multimon-ng](https://github.com/EliasOenal/multimon-ng) decodes many paging, AFSK modes, and similar, including POCSAG, FLEX, EAS.
-+ [Codec2](https://github.com/drowe67/codec2) is a FOSS ultra-low bitrate voice codec that's competitive from 3200 bits/s down to 450 bits/s. Also includes [FreeDV](https://freedv.org/), an HF digital voice mode. [PyCodec2](https://github.com/gregorias/pycodec2) is an easy Python wrapper.
++ [Codec2](https://github.com/drowe67/codec2) is a FOSS ultra-low bitrate voice codec that's competitive from 3200 bits/s down to 450 bits/s.
+  + [FreeDV](https://freedv.org/), an HF digital voice mode based upon Codec2.
```
Comment (DD5HT) [inline README.md:35]: Extra whitespace?

**Review by DD5HT — APPROVED** [review]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (kd9lsv) [reply CONTRIBUTING.md:12]: Depends. It also means users have to have a GitHub account as well to contribute. 

I am okay with it being removed too, but just allowing options to submit data.

File: README.md (hunk 2 above)
Comment (kd9lsv) [reply README.md:35]: No, the items below are based upon Codec2 as applications.

File: CONTRIBUTING.md (hunk 1 above)
Comment (DD5HT) [reply CONTRIBUTING.md:12]: Let's just keep it for now then.

**Pull request merged**